app/org/sagebionetworks/bridge/play/controllers/ParticipantController.java

```diff
@@ -95,34 +101,55 @@ public Result updateSelfParticipant() throws Exception {
         
         return getParticipantsInternal(study, offsetByString, pageSizeString, emailFilter, startDateString,
-                endDateString);
+                endDateString, startTimeString, endTimeString);
     }
 
     public Result getParticipantsForWorker(String studyId, String offsetByString, String pageSizeString, String emailFilter,
-            String startDateString, String endDateString) {
+            String startDateString, String endDateString, String startTimeString, String endTimeString) {
         getAuthenticatedSession(WORKER);
         
         Study study = studyService.getStudy(studyId);
         return getParticipantsInternal(study, offsetByString, pageSizeString, emailFilter, startDateString,
-                endDateString);
+                endDateString, startTimeString, endTimeString);
     }
     
     private Result getParticipantsInternal(Study study, String offsetByString, String pageSizeString,
-            String emailFilter, String startDateString, String endDateString) {
+            String emailFilter, String startDateString, String endDateString, String startTimeString,
+            String endTimeString) {
         
         int offsetBy = getIntOrDefault(offsetByString, 0);
         int pageSize = getIntOrDefault(pageSizeString, API_DEFAULT_PAGE_SIZE);
-        DateTime startDate = DateUtils.getDateTimeOrDefault(startDateString, null);
-        DateTime endDate = DateUtils.getDateTimeOrDefault(endDateString, null);
         
+        // For naming consistency, we are changing from the user of startDate/endDate to startTime/endTime
+        // for DateTime parameters. Both are accepted by these participant API endpoints (the only places 
+        // where this needed to change).
+        DateTime startTime = DateUtils.getDateTimeOrDefault(startTimeString, null);
+        if (startTime == null) {
+            startTime = DateUtils.getDateTimeOrDefault(startDateString, null);
+        }
+        DateTime endTime = DateUtils.getDateTimeOrDefault(endTimeString, null);
+        if (endTime == null) {
+            endTime = DateUtils.getDateTimeOrDefault(endDateString, null);
+        }
         PagedResourceList<AccountSummary> page = participantService.getPagedAccountSummaries(study, offsetBy, pageSize,
-                emailFilter, startDate, endDate);
-        return okResult(page);
+                emailFilter, startTime, endTime);
+        
+        // Similarly, we will return startTime/endTime in the top-level request parameter properties as 
+        // startDate/endDate while transitioning, to maintain backwards compatibility.
+        ObjectNode node = (ObjectNode)MAPPER.valueToTree(page);
+        Map<String,Object> rp = page.getRequestParams();
+        if (rp.get(START_TIME) != null) {
+            node.put(START_DATE, (String)rp.get(START_TIME));    
+        }
+        if (rp.get(END_TIME) != null) {
+            node.put(END_DATE, (String)rp.get(END_TIME));    
+        }
```

Comment:
I feel like these older values are deprecated, so when we remove startDate/endDate from the routes/controller, it'll be easier to find and remove this code. If you feel strongly about it, I can't move it.